docs/authentication.md

```diff
@@ -0,0 +1,24 @@
+# Authentication
+
+VolView itself does not specify any given authentication mechanism to use. How you let VolView access authenticated resources depends on individual deployment needs.
+
+## VolView behind Login Portal
+
+The easiest approach to securing a VolView instance is to host VolView behind an authentication portal. This will secure access to VolView, and allow VolView to automatically use existing user credentials to request protected data.
```

Comment:
```suggestion
The easiest approach to securing a VolView instance is to host VolView behind an authentication portal. This will secure access to VolView, and allow VolView to automatically use existing user credentials to request protected data.  Often the credentals are stored in a cookie.
```
Are there other common ways authentical portal's transparently include credentials in fetch requests?

Reply:
For fetch requests, there's other way to store tokens, such as local storage or in-memory.